components/textfield/metadata/textarea.yml

```diff
@@ -8,37 +8,124 @@ sections:
      See the [Textfield migration guide](textfield.html#migrationguide) for migration information.
 examples:
   - id: textfield
-    name: Standard
+    name: Standard Sizes
+    markup: |
+      <div class="spectrum-Textfield spectrum-Textfield--sizeS spectrum-Textfield--multiline">
```

Comment:
The layout for the examples here feels a little tight. Maybe the following added to the example wrapper?

```
display: inline-flex;
gap: 12px;
flex-wrap: wrap;
```